source/GameData.cpp

```diff
@@ -99,7 +99,8 @@ namespace {
 	const Government *playerGovernment = nullptr;
 	map<const System *, map<string, int>> purchases;
 
-	ConditionsStore globalConditions;
+	shared_ptr<ConditionsStore> globalConditions = make_shared<ConditionsStore>();
+	shared_ptr<ConditionsStore> varyingConditions = make_shared<ConditionsStore>(); // AKA: PlayerInfo.conditions
```

Comment:
Some conditions are really player specific (like which planets did the player visit? What is the players reputation?), while other conditions are more "universe specific" (like "did some war already start?"). Moving the conditions from PlayerInfo to GameData (or UniverseObjects) could be an improvement for "universe-related conditions", while it feels like a step backwards for the conditions specific to a player.
I'm not sure though how to handle this best... (Since we currently don't differentiate between those types of conditions.)

Maybe we should introduce some mechanisms (in a different PR) to differentiate between those types of conditions? (And store the player-specific conditions in PlayerInfo, while storing the universe-specific conditions here?)

Reply:
I see two ways to deal with that:

1. All universe conditions must begin with something specific, like `universe: ` and PlayerInfo.conditions has a derived provider that connects it to the universe conditions.
2. Scopes. Anything that isn't in PlayerInfo.conditions can come from the universe conditions, but the code must explicitly set universe conditions, perhaps with `universe "condition name" += 7` in actions.

Reply:
You should comment about this in #8066 which PRs just this one feature.

Reply:
I don't think it'd be good to differentiate both conditions because then we wouldnt be able to use all of them here?

Reply:
I think we should leave this problem to a follow-up PR rather than making this PR more complicated.